addons/website_forum/static/src/js/website_forum.share.js

```diff
@@ -24,31 +23,22 @@ var ForumShare = publicWidget.registry.socialShare.extend({
      */
     start: function () {
         var def = this._super.apply(this, arguments);
-        this._onMouseEnter();
-        return def;
-    },
-
-    //--------------------------------------------------------------------------
-    // Private
-    //--------------------------------------------------------------------------
-
-    /**
-     * @private
-     */
-    _render: function () {
         var $question = this.$('article.question');
         if (!this.targetType) {
             this._super.apply(this, arguments);
-        } else if (this.targetType === 'social-alert') {
-            $question.before(renderToElement('website.social_alert', {medias: this.socialList}));
         } else {
-            $('body').append(renderToElement('website.social_modal', {
-                medias: this.socialList,
+            const el = renderToElement('website.social_modal', {
```

Comment:
Is there a point to render it in edit mode? It's not clickable anyway to customize it, it just closes itself.
Making it customizable might open door to other problems (what if user remove everything etc), but I'd at least make it user friendly in edit mode.

Reply:
Maybe there is a confusion: this render is about the modal that appears when creating a new post - not about the inline share button.
I.e.: this is what is rendered here:
![image](https://github.com/odoo/odoo/assets/71644421/7176224f-02fb-4bb6-9179-856b41fa5d88)

Reply:
Indeed, thank you for the precision

Reply:
Added `css_editable_mode_hidden` on the dropdown menu so that even if it gets opened, nothing is displayed during edit.